dependencies.props

```diff
@@ -169,6 +169,7 @@
 
   <PropertyGroup>
     <XUnitPackageVersion>2.3.0-beta1-build3642</XUnitPackageVersion>
+    <CompatibilityShimsPackageVersion>2.1.0</CompatibilityShimsPackageVersion>
```

Comment:
@danmosemsft @terrajobst do we want this to start at 2.0.0?

Reply:
I think so, it has no relationship to 2.1